### ISSUE TITLE
show download link (icon) on stored page

### ISSUE THREAD

**Issue by significance** [fairDataSociety/Fairdrop#211]

**Comment by plur9** [comment]
not sure what this is? Is this the "download page"?

**Comment by significance** [comment]
yes, i guess it's easy to work out that you click on the filename to start the download, but i think as we add different actions eg. pin/unpin then we will migrate to having action buttons on the right hands side also.
